partner_attribute_hls/models/res_partner.py

```diff
@@ -0,0 +1,9 @@
+# Copyright 2021 Quartile Limited
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    customer_code = fields.Char(copy=False)
```

Comment:
```suggestion
    customer_code = fields.Char(copy=False, help="Intended to be passed to the external accounting system.")
```
On second thought, it might be better to rename the field to `partner_code`, given the possibility of passing the payable records to the accounting system.  But I'm OK with the current name as well.